src/main/java/de/zalando/aruha/nakadi/controller/SubscriptionController.java

```diff
@@ -55,52 +55,70 @@ public SubscriptionController(final SubscriptionDbRepository subscriptionReposit
 
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<?> createOrGetSubscription(@Valid @RequestBody final SubscriptionBase subscriptionBase,
-                                                     final Errors errors, final NativeWebRequest nativeWebRequest) {
-
-        if (!featureToggleService.isFeatureEnabled(HIGH_LEVEL_API)) {
+                                                     final Errors errors,
+                                                     final NativeWebRequest request) {
+        if (!featureToggleService.isFeatureEnabled(FeatureToggleService.Feature.HIGH_LEVEL_API)) {
```

Comment:
I refactored this class a bit as a part of my investigation of implementation of that feature. I agree that refactor all the code and fix all static import does not make sense.